content/logs/index.md

```diff
@@ -147,22 +194,22 @@ However, if your logs defines one of the following attributes Datadog considers
 * `eventTime`
 * `published_date`
 
-You can also ask Datadog to consider any attribute as the official date reference thanks to the [log date remapper processor](/logs/processing/#log-date-remapper)
+You can also ask Datadog to interpret the value of any other attribute as the log’s official date if you set a [log date remapper processor](/logs/processing/#log-date-remapper)
 <div class="alert alert-info">
 The recognized date formats are: <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO8601</a>, <a href="https://en.wikipedia.org/wiki/Unix_time">UNIX (the milliseconds EPOCH format)</a>  and <a href="https://www.ietf.org/rfc/rfc3164.txt">RFC3164</a>.
 </div>
 
 ### `message` attribute
 
-By default, Datadog considers the `message` as the main content of the log. It has then a special display in the [log list](/logs/explore/#log-list) and you can run full text search on it.
+By default, Datadog considers the `message` value as the main content of the log. That value will then have a special display in the [log list](/logs/explore/#log-list) and you can run full text search on it.
 
-The `message` attribute is indexed as text for the [search bar](/logs/explore/#search-bar) allowing you to search over any tokenized word without mentioning the path to the attribute.
+The `message` attribute is indexed as text for the [search bar](/logs/explore/#search-bar), allowing you to search over any tokenized word without mentioning the path to the attribute.
 
 ### `severity` attribute
 
-Each log has its own severity, and your datadog application levrage a lot this information.
+Each log has its own severity, and your datadog application will levrage a lot with this information.
```

Comment:
"Datadog" rather than "datadog".
What does "leverage a lot mean" ?
"Levrage" is misspelled.

Maybe:
"Each log entry may specific a severity level which will be made available for faceted search within Datadog"

Reply:
👍